src/main/java/com/google/sps/servlets/UserStatusServlet.java

```diff
@@ -33,17 +33,14 @@
   // Response: returns some user information JSON from the current user's sign-in API profile.
   // Data: "isUserLoggedIn" : boolean
   @Override
-  public void doGet (HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
-    UserService userService = UserServiceFactory.getUserService();
 
     JsonObject userInfo = new JsonObject();
-    userInfo.addProperty("isUserLoggedIn", userService.isUserLoggedIn());
-    if (userInfo.get("isUserLoggedIn").getAsBoolean()) {
+    userInfo.addProperty("isUserLoggedIn", UserHelper.isUserLoggedIn());
+    if (UserHelper.isUserLoggedIn()) {
         System.out.println("User is logged in");
```

Comment:
is this if statement supposed to be here or was it just a debugging thing?

Reply:
I might return info from the google sign-in API's user profile some time in the future, so this if statement is a place-holder.

Reply:
I've changed it to reflect that